utilities/class.tapestry-helpers.php

```diff
@@ -219,6 +219,12 @@ public static function userIsAllowed($action, $nodeMetaId, $tapestryPostId, $sup
         $groupIds = self::getGroupIdsOfUser($userId, $tapestryPostId);
         $user = new TapestryUser($userId);
 
+        // If user is author and node is in review, user is not allowed to edit node
+        $inReview = isset($node->reviewStatus);
+        if ($action === "EDIT" && $inReview && $user->isAuthorOfThePost($nodePostId)) {
```

Comment:
I'd replace `$user->isAuthorOfThePost($nodePostId)` with `!$user->canEdit($tapestryPostId)` since super admins should still be able to edit nodes, regardless of whether they are the author or not.
```suggestion
        if ($action === "EDIT" && $inReview && !$user->canEdit($tapestryPostId)) {
```

Reply:
@wynnset Since this is returning false for isUserAllowed, if I change it to canEdit, it means that a user who is editing a node in review **cannot** edit the node.

Reply:
@shirleywbi notice the `!` so it is a negative check. It means if the user canNOT edit the tapestry, it will return false. If the user can edit the tapestry, that conditional return won't execute.